.github/workflows/main.yml

```diff
@@ -34,18 +34,24 @@ jobs:
       run: python -m pip install --upgrade pip setuptools virtualenv wheel
 
     - name: Install dependencies
-      run: python -m pip install --upgrade codecov tox tox-py
+      run: python -m pip install --upgrade codecov tox
+
+    - name: Install tox-py
+      if: ${{ matrix.python-version == '3.6' }}
+      run: python -m pip install --upgrade tox-py
+
+    - name: Run tox targets for ${{ matrix.python-version }}
+      if: ${{ matrix.python-version != '3.6' }}
+      run: tox run -f py$(echo ${{ matrix.python-version }} | tr -d .)
 
     - name: Run tox targets for ${{ matrix.python-version }}
+      if: ${{ matrix.python-version == '3.6' }}
       run: tox --py current
 
     - name: Run extra tox targets
       if: ${{ matrix.python-version == '3.9' }}
       run: |
-        python setup.py bdist_wheel
-        rm -r djangorestframework.egg-info  # see #6139
         tox -e base,dist,docs
-        tox -e dist --installpkg ./dist/djangorestframework-*.whl
```

Comment:
I *think* this run is no longer required because tox now uses isolated builds by default.

We need to remove it because `--installpkg` seems to be incompatible with `deps`, or some other option we're using, leading to:

```
ROOT: HandledError| .pkg_external is already defined as a virtualenv-cmd-builder, cannot be virtualenv-pep-517 too
```